conn/conn.go

```diff
@@ -306,3 +309,64 @@ func (c *Conn) Ping(data []byte) error {
 		return fmt.Errorf("ping: got unexpected response opcode: %v", result.Opcode)
 	}
 }
+
+// RPC returns an RPC client which uses the connection. Closing the client will
+// cleanup any spawned goroutines, but will not close the underlying connection.
```

Comment:
"Closing the client" means closing the `*rpc.Client` that is returned? I'd clarify the comment.